config/crd/bases/topology.nephio.org_nftopologies.yaml

```diff
@@ -165,6 +186,48 @@ spec:
             type: object
           status:
             description: NFTopologyStatus defines the observed state of NFTopology
+            properties:
+              conditions:
+                description: Current service state of the NF.
+                items:
+                  type: string
+                type: array
+              nfInstances:
+                description: Detail on the deployed instances.
+                items:
+                  description: NFDeployedInstance defines an NF instance that is deployed
+                  properties:
+                    clusterName:
+                      description: name of workload cluster where the NF instance
+                        is to be deployed
+                      type: string
+                    connectivities:
+                      description: "TODO(s3wong): the following parameters should
+                        eventually be metadata embedded in a NF package (NFType should
+                        be as well) // NF vendor name NFVendor string `json:\"nfVendor,omitempty\"
+                        yaml:\"nfVendor,omitempty\"` // the software version of this
+                        NF vendor's NFType NFVersion string `json:\"nfVersion,omitempty\"
+                        yaml:\"nfVersion,omitempty\"` \n list of connected NF instances
+                        to this NF instance"
+                      items:
+                        properties:
+                          neighborName:
+                            description: peer NF's Id (see NFInstance struct below)
```

Comment:
Is this needed here?
(see NFInstance struct below)

Reply:
Yes --- this is the list of neighbor IDs, which is a simple way to record a set of connected NFs (according to how the template is instantiated). NFInstance is just a template --- every time a cluster is created with the specified labels, an instance of NF matching this template is deployed onto the cluster, and that instance forms a neighbor relationship on the existing NFs that aren't part of this NFInstance template but has attachment on the same network.